diff --git a/base/BUILD.gn b/base/BUILD.gn
index e0f8cce34cbffb5853abffdc3267f685f73e716a..ef41e558370cdaf98fb906e85fc240fcdb53ddeb 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5081,6 +5081,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5755,6 +5756,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
