diff --git a/base/feature_list.h b/base/feature_list.h
index 0a16cbcedd025eee414cd869e6b56f0dbb2f5e56..322828825ed476a887e436bc4e0e7900df83111c 100644
--- a/base/feature_list.h
+++ b/base/feature_list.h
@@ -95,6 +95,11 @@ class BASE_EXPORT FeatureList {
   FeatureList& operator=(const FeatureList&) = delete;
   ~FeatureList();
 
+  bool IsFeatureOverridden_ChromiumImpl(std::string_view feature_name) const;
+  static std::optional<bool> GetStateIfOverridden_ChromiumImpl(
+      const Feature& feature);
+  static FeatureState GetCompileTimeFeatureState(const Feature& feature);
+
   // Used by common test fixture classes to prevent abuse of ScopedFeatureList
   // after multiple threads have started.
   class BASE_EXPORT ScopedDisallowOverrides {
