diff --git a/base/feature_list.cc b/base/feature_list.cc
index 1b4810d0a53526c47d132b4f5fba918dce6ffd8a..86f8cf04a4b1c86bca69b4e1b8fe096e7b4bc0f9 100644
--- a/base/feature_list.cc
+++ b/base/feature_list.cc
@@ -623,7 +623,7 @@ bool FeatureList::UpdateRuntimeMutableFeatureState(
   return true;
 }
 
-bool FeatureList::IsFeatureOverridden(std::string_view feature_name) const {
+bool FeatureList::IsFeatureOverridden_ChromiumImpl(std::string_view feature_name) const {
   return GetOverrideEntryByFeatureName(feature_name);
 }
 
@@ -730,8 +730,8 @@ bool FeatureList::IsEnabled(const Feature& feature) {
     EarlyFeatureAccessTracker::GetInstance()->AccessedFeature(
         feature, g_feature_list_instance &&
                      g_feature_list_instance->IsEarlyAccessInstance());
-    return feature.default_state == FEATURE_ENABLED_BY_DEFAULT ||
-           feature.default_state == FEATURE_DISABLED_FOR_COUNTRIES;
+    return GetCompileTimeFeatureState(feature) == FEATURE_ENABLED_BY_DEFAULT ||
+           GetCompileTimeFeatureState(feature) == FEATURE_DISABLED_FOR_COUNTRIES;
   }
   return g_feature_list_instance->IsFeatureEnabled(feature);
 }
@@ -742,7 +742,7 @@ bool FeatureList::IsValidFeatureOrFieldTrialName(std::string_view name) {
 }
 
 // static
-std::optional<bool> FeatureList::GetStateIfOverridden(const Feature& feature) {
+std::optional<bool> FeatureList::GetStateIfOverridden_ChromiumImpl(const Feature& feature) {
   if (!g_feature_list_instance ||
       !g_feature_list_instance->AllowFeatureAccess(feature)) {
     EarlyFeatureAccessTracker::GetInstance()->AccessedFeature(
@@ -1054,7 +1054,7 @@ bool FeatureList::IsFeatureEnabled(const Feature& feature) const {
     return overridden_state == OVERRIDE_ENABLE_FEATURE;
   }
 
-  if (IsCountrySpecificFeatureState(feature.default_state)) [[unlikely]] {
+  if (IsCountrySpecificFeatureState(GetCompileTimeFeatureState(feature))) [[unlikely]] {
     const auto& restricted_feature =
         static_cast<const FeatureWithCountryRestriction&>(feature);
     switch (restricted_feature.default_state) {
@@ -1069,7 +1069,7 @@ bool FeatureList::IsFeatureEnabled(const Feature& feature) const {
     }
   }
 
-  return feature.default_state == FEATURE_ENABLED_BY_DEFAULT;
+  return GetCompileTimeFeatureState(feature) == FEATURE_ENABLED_BY_DEFAULT;
 }
 
 std::optional<bool> FeatureList::IsFeatureEnabledIfOverridden(
