diff --git a/chrome/browser/ui/webui/unexportable_keys_internals/unexportable_keys_internals_handler.cc b/chrome/browser/ui/webui/unexportable_keys_internals/unexportable_keys_internals_handler.cc
index 2e75705c04f983a16a4b27aaa43983e673a9d731..e3d725286ebfce45ea4a2c29592136a60a639fb3 100644
--- a/chrome/browser/ui/webui/unexportable_keys_internals/unexportable_keys_internals_handler.cc
+++ b/chrome/browser/ui/webui/unexportable_keys_internals/unexportable_keys_internals_handler.cc
@@ -32,6 +32,8 @@ std::string_view GetAlgorithmName(
       return "ECDSA_SHA256";
     case crypto::SignatureVerifier::SignatureAlgorithm::RSA_PSS_SHA256:
       return "RSA_PSS_SHA256";
+    case crypto::SignatureVerifier::SignatureAlgorithm::ECDSA_SHA384:
+      return "ECDSA_SHA384";
   }
 }
 
