diff --git a/chrome/android/chrome_public_apk_tmpl.gni b/chrome/android/chrome_public_apk_tmpl.gni
index b2d240ef7b153c943ead76a71287d82c791944d9..e8196257665307cfe52720c499ec4f719373836b 100644
--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -297,6 +297,7 @@ template("chrome_common_apk_or_module_tmpl") {
       "*ic_file_download_white*",  # Bottom edge seems misaligned.
       "*ic_lock.*",  # Bottom edge seems misaligned.
     ]
+    resource_exclusion_exceptions += brave_resources_exclusion_exceptions
 
     # Most of these, with the exception of resource_exclusion_exceptions,
     # are currently duplicated in system_webview_apk_tmpl.gni.
@@ -319,6 +320,7 @@ template("chrome_common_apk_or_module_tmpl") {
 
       proguard_enabled = true
       proguard_configs = [ "//chrome/android/proguard/main.flags" ]
+      proguard_configs += [ "//brave/android/java/proguard.flags" ]
       if (_is_test) {
         proguard_configs += [ "//chrome/android/proguard/apk_for_test.flags" ]
       }
@@ -656,6 +658,7 @@ template("chrome_common_apk_or_module_tmpl") {
                                  "version_code",
                                  "version_name",
                                ])
+    _wireguard_target = "//brave/third_party/android_deps:com_wireguard_android_J__unpack_aar" deps += [ _wireguard_target] _libwireguard_dir = get_label_info("//brave/third_party/android_deps:com_wireguard_android($default_toolchain)","target_out_dir") + "/com_wireguard_android_java/jni" loadable_modules +=[ "$_libwireguard_dir/$android_app_abi/libwg-go.so" ]
   }
 }
 
