diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 783af1b22c5397da02570a0e2de33d864d24dba8..902fd721f9f408c58a4db281e0869c68f236c53e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4725,6 +4725,7 @@ const FeatureEntry::FeatureVariation kToolbarGlowUpVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
