diff --git a/chrome/browser/storage_access_api/storage_access_grant_permission_context.cc b/chrome/browser/storage_access_api/storage_access_grant_permission_context.cc
index 18b4fdcee63350f4fa181fd1724a4778682e7668..3a9955d4d421c13410447f4eba34ea9870c2d7c1 100644
--- a/chrome/browser/storage_access_api/storage_access_grant_permission_context.cc
+++ b/chrome/browser/storage_access_api/storage_access_grant_permission_context.cc
@@ -464,6 +464,7 @@ void StorageAccessGrantPermissionContext::CheckForAutoGrantOrAutoDenial(
         break;
     }
   }
+  BRAVE_STORAGE_ACCESS_GRANT_PERMISSION_CONTEXT_CHECK_FOR_AUTO_GRANT_OR_AUTO_DENIAL  // NOLINT
 
   // Get all of our implicit grants and see which ones apply to our
   // |requesting_origin|.
