diff --git a/chrome/browser/ui/views/user_education/browser_user_education_service.cc b/chrome/browser/ui/views/user_education/browser_user_education_service.cc
index fec26d22bb4f90c615075fbb301195601bcd83b0..7b89876260f6952eb3d9067ab52cb2f0a903c247 100644
--- a/chrome/browser/ui/views/user_education/browser_user_education_service.cc
+++ b/chrome/browser/ui/views/user_education/browser_user_education_service.cc
@@ -364,6 +364,7 @@ void MaybeRegisterChromeFeaturePromos(
           feature_engagement::kIPHWebUiHelpBubbleTestFeature)) {
     return;
   }
+  BRAVE_MAYBE_REGISTER_CHROME_FEATURE_PROMOS
 
   // kIPHAutofillCreditCardBenefitFeature:
   registry.RegisterFeature(std::move(
