diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index f73aae4b10cc409c8a84b5d97fcb8a0f371bc58f..1b7b61dacc1f1dd2b226458bb8b86b29e026ffb2 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -4604,6 +4604,7 @@ source_set("browser") {
   if (!is_android) {
     deps += [ "//chrome/browser/contextual_cueing/internals" ]
   }
+  public_deps += [ "//brave/browser:browser_impl" ]
 
   # This target intentionally has no sources of its own and no
   # allow_circular_includes_from; it only aggregates dependencies. Do NOT add
@@ -4774,6 +4775,8 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni")
+  public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
