diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index aa04f15bff66a4b6440172cf7104ac223217cca9..a931ea826744e1fe3c2e490e9594f74a5f3950eb 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4783,6 +4783,7 @@ const FeatureEntry::FeatureVariation kGlassFrameVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
