diff --git a/chrome/browser/browsing_data/BUILD.gn b/chrome/browser/browsing_data/BUILD.gn
index 4f97f5cbef8cf42d7bff28fda0d1c7debcd95ab1..87fc950ebd07ccf75d3bb8b6171ab1051f57a484 100644
--- a/chrome/browser/browsing_data/BUILD.gn
+++ b/chrome/browser/browsing_data/BUILD.gn
@@ -12,6 +12,7 @@ source_set("constants") {
     "//content/public/browser",
     "//extensions/buildflags",
   ]
+  public_deps = [ "//brave/components/ai_chat/core/common/buildflags" ]
 }
 
 source_set("browsing_data") {
