diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 3b8d9154e22a3116dc6392e4ef8e83c0d2c7dafa..e1cbffae958e9d74b8725fdf4c4cd17d3867ec2c 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/extensions/resources.gni")
 import("//build/config/chromebox_for_meetings/buildflags.gni")
 import("//build/config/features.gni")
 import("//build/config/ozone.gni")
@@ -649,6 +650,7 @@ source_set("extensions") {
     "//ui/native_window_tracker",
     "//ui/shell_dialogs",
   ]
+  deps += brave_extensions_resources
   public_deps = [
     "//chrome/browser/extensions/api/messaging",
     "//chrome/browser/media/webrtc",
