diff --git a/chrome/browser/ui/unload_controller.cc b/chrome/browser/ui/unload_controller.cc
index 6c6d988d0f4ded4f82e8dc9ec9c26f483bfe9cf1..f1aaa79f838bbc17814eae47560e0e602fb714d5 100644
--- a/chrome/browser/ui/unload_controller.cc
+++ b/chrome/browser/ui/unload_controller.cc
@@ -450,6 +450,7 @@ UnloadController::GetBrowserClosingStatus() {
 bool UnloadController::TryToCloseWindow(
     bool skip_beforeunload,
     const base::RepeatingCallback<void(bool)>& on_close_confirmed) {
+  confirmed_to_close_ = true;
   cancel_download_confirmation_state_ =
       CancelDownloadConfirmationState::kResponseReceived;
   // The devtools browser gets its beforeunload events as the results of
@@ -472,6 +473,7 @@ bool UnloadController::TryToCloseWindow(
 }
 
 void UnloadController::ResetTryToCloseWindow() {
+  confirmed_to_close_ = false;
   cancel_download_confirmation_state_ =
       CancelDownloadConfirmationState::kNotPrompted;
   if (!is_calling_before_unload_handlers()) {
@@ -570,6 +572,9 @@ void UnloadController::OnTabStripModelChanged(
 }
 
 void UnloadController::TabStripEmpty() {
+  if (!browser_->profile()->GetPrefs()->GetBoolean(kEnableClosingLastTab)) {
+    return;
+  }
   // Set is_attempting_to_close_browser_ here, so that extensions, etc, do not
   // attempt to add tabs to the browser before it closes.
   is_attempting_to_close_browser_ = true;
@@ -921,6 +926,9 @@ void UnloadController::InProgressDownloadResponse(bool cancel_downloads) {
 }
 
 void UnloadController::FinishWarnBeforeClosing(WarnBeforeClosingResult result) {
+  if (result == WarnBeforeClosingResult::kDoNotClose) {
+    confirmed_to_close_ = false;
+  }
   switch (result) {
     case WarnBeforeClosingResult::kOkToClose:
       chrome::CloseWindow(browser_);
