diff --git a/components/sync/protocol/entity_specifics.proto b/components/sync/protocol/entity_specifics.proto
index 9b40f1908e2793ad0557a9b96d5aaa55b5466242..179777af1c7e8e43556a4dd1675d6b6a1ef462d6 100644
--- a/components/sync/protocol/entity_specifics.proto
+++ b/components/sync/protocol/entity_specifics.proto
@@ -11,6 +11,7 @@ option optimize_for = LITE_RUNTIME;
 
 package sync_pb;
 
+import "brave/components/sync/protocol/ai_chat_specifics.proto";
 import "components/sync/protocol/ai_thread_specifics.proto";
 import "components/sync/protocol/contextual_task_specifics.proto";
 import "components/sync/protocol/encryption.proto";
@@ -177,6 +178,7 @@ message EntitySpecifics {
     GeminiThreadSpecifics gemini_thread = 1569348;
     EncryptedTabContextContainerSpecifics encrypted_tab_context_container =
         1656053;
+    AIChatConversationSpecifics ai_chat_conversation = 10000001;
     // When adding a new type, follow the docs below and keep this comment as
     // the last entry.
     // https://www.chromium.org/developers/design-documents/sync/integration-checklist/
