diff --git a/components/sync/engine/cycle/data_type_tracker.cc b/components/sync/engine/cycle/data_type_tracker.cc
index 2531b0bddf07e42e54572c6160fdebc4eb58db74..28aeff8c25d9c9c8f1f4ed5ba8b59eb1a6766d87 100644
--- a/components/sync/engine/cycle/data_type_tracker.cc
+++ b/components/sync/engine/cycle/data_type_tracker.cc
@@ -120,6 +120,7 @@ base::TimeDelta GetDefaultLocalChangeNudgeDelay(DataType data_type) {
     case SKILL:
     case GEMINI_THREAD:
     case THEMES_ANDROID:
+    case AI_CHAT_CONVERSATION:
       return kMediumLocalChangeNudgeDelay;
     case UNSPECIFIED:
       NOTREACHED();
@@ -199,6 +200,7 @@ bool CanGetCommitsFromExtensions(DataType data_type) {
     case SKILL:
     case GEMINI_THREAD:
     case THEMES_ANDROID:
+    case AI_CHAT_CONVERSATION:
       return false;
     case UNSPECIFIED:
       NOTREACHED();
