diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index 36ac7df8a0a6b87cf073d73cf2f91fae2db0dfb4..2c2c6f55bb1f9d298a4c398a133eb34f7d810967 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -357,6 +357,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
