diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 96f7c10555294c3f48fd81d7f4d0ad779bdd7692..f6f18f4f18260c801ad644de1ab5d8770f6bf8b0 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -64,6 +64,7 @@ source_set("common") {
   # must not link to this because it will duplicate the code in the component
   # build.
   visibility = [ "//content/*" ]
+  visibility += [ "//brave/content/*" ]
 
   sources = [
     "background_fetch/background_fetch_types.cc",
