diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 69d4174e7c7f204a3de846966fd9a0898a7a5eea..2ffb929049032ad9e44915e31d53f7aebb10fbd5 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3850,6 +3850,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
