diff --git a/DEPS b/DEPS
index efe5708584e2767bf8e691d97b42a3195c3e732e..f3a95ce96e0c486be8e6fab21ac3eddef4d0ecfe 100644
--- a/DEPS
+++ b/DEPS
@@ -4420,7 +4420,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
