diff --git a/DEPS b/DEPS
index f728479528dea8f8e702b6b8dde6ab7d8f25de79..1d78cc8bd1c83da1cca8796bb54c7ef8434f217b 100644
--- a/DEPS
+++ b/DEPS
@@ -4375,7 +4375,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
