diff --git a/net/BUILD.gn b/net/BUILD.gn
index e2e399a8b78d00a587bdc23e396b447b113531bc..e85dec82c15f1a7f5e5663154f4f782c850608a6 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1288,6 +1288,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
