diff --git a/third_party/blink/renderer/core/dom/document.cc b/third_party/blink/renderer/core/dom/document.cc
index a784adf9ec7596bdd4d4c1c30516aba47b2838b2..e42db4bead31b270afcb3fdc20d27687be2ef367 100644
--- a/third_party/blink/renderer/core/dom/document.cc
+++ b/third_party/blink/renderer/core/dom/document.cc
@@ -7576,6 +7576,7 @@ KURL Document::CompleteURLWithOverride(
   if (result.IsEmpty()) {
     result = Encoding().IsValid() ? KURL(base_url_override, url, Encoding())
                                   : KURL(base_url_override, url);
+    result = RewriteBraveToChrome(result);
     url_cache_.Put(base_url_override, url, result);
   }
   // If the conditions are met for
