diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 8e20b6dc21c02da9e497fb4ce823af8a51ae0a91..6b321626df9a39221057d5401a121271adfd2445 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2860,6 +2860,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
