diff --git a/front_end/panels/application/ApplicationPanelSidebar.ts b/front_end/panels/application/ApplicationPanelSidebar.ts
index 22d584a59c8de7f326a6c4f664e2c7a2e9b5e460..bbd34824a8f483fc05a98b5a61bd3d412c02d3ac 100644
--- a/front_end/panels/application/ApplicationPanelSidebar.ts
+++ b/front_end/panels/application/ApplicationPanelSidebar.ts
@@ -607,6 +607,7 @@ export class ApplicationPanelSidebar extends UI.Widget.VBox implements SDK.Targe
         new Common.ObjectWrapper.ObjectWrapper<SharedStorageTreeElementDispatcher.EventTypes>();
 
     this.contentElement.style.contain = 'layout style';
+    [this.trustTokensTreeElement, this.interestGroupTreeElement, this.sharedStorageListTreeElement].forEach(e => storageTreeElement.removeChild(e)); [this.backgroundFetchTreeElement, this.backgroundSyncTreeElement, this.periodicBackgroundSyncTreeElement, this.reportingApiTreeElement].forEach(e => backgroundServiceTreeElement.removeChild(e));
   }
 
   private addSidebarSection(title: string, jslogContext: string): UI.TreeOutline.TreeElement {
