diff --git a/front_end/core/sdk/NetworkManager.ts b/front_end/core/sdk/NetworkManager.ts
index e4b705cfb4f13e64ee805cc725beabc841b079c0..cf1c5d8e6dfe481892cac00c0a16a02713a8a481 100644
--- a/front_end/core/sdk/NetworkManager.ts
+++ b/front_end/core/sdk/NetworkManager.ts
@@ -1647,6 +1647,7 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
     return `${host}:${port}`;
   }
 }
+import { PatchNetworkDispatcher } from './NetworkManager.patch.js'; (NetworkDispatcher as any) = PatchNetworkDispatcher(NetworkDispatcher);
 
 export type RequestConditionsSetting = {
   url: string,
