diff --git a/BUILD.gn b/BUILD.gn
index 249dbe7f219ad13f978f885711471292f81fc188..007b2275f5b48b3678c12a9304c6b8fa9b30ff81 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -910,6 +910,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3596,6 +3597,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
